mindmeld/server.py

```diff
@@ -140,11 +127,9 @@ def handle_server_error(error):
             logger.error(json.dumps(response_data))
             return response
 
-        @server.route("/_status", methods=["GET"])
+        @server.route("/health", methods=["GET"])
```

Comment:
Are we sure no downstream services might be affected by this change? If people were using the `_status` endpoint before, it will no longer work once the lib is updated.